frontend/src/concepts/pipelines/content/createRun/RunPageFooter.tsx

```diff
@@ -13,6 +18,7 @@ type RunPageFooterProps = {
 
 const RunPageFooter: React.FC<RunPageFooterProps> = ({ data, contextPath }) => {
   const { api } = usePipelinesAPI();
+  const { runType } = useGetSearchParamValues([PipelineRunSearchParam.RunType]);
```

Comment:
Yup, this is a bug.
![image](https://github.com/opendatahub-io/odh-dashboard/assets/8126518/b5a11691-643b-4906-adfd-1a44032497d0)

Reply:
@andrewballantyne Could we remove the "Run type" section? When selecting to create an "active run", users actually select the type. Right?

Reply:
Jeff pinged me about this -- I responded in slack not on the thread.

For posterity: "I don’t mind which direction we go — I just don’t want the inconsistency / bug of the current functionality".

If we want to make these create screen "one way or the other" from the start -- I don't mind. I imagine this is just a flag we can statically hold at the top based on the route or something. Shouldn't need two full components @jpuzz0 (not sure if you have already coded this or not)

Reply:
@andrewballantyne in my last commit, I addressed this by removing the "run type" section as suggested by @xianli123.

Reply:
Good call @Gkrumbach07. I forgot about that. Just updated.

Reply:
> @andrewballantyne in my last commit, I addressed this by removing the "run type" section as suggested by @xianli123.

Sorry for chiming in late. If we remove this `Run type` option, we need add `Create schedule `button at the Pipeline action button under the Pipeline table list and detail page. We only have `Create run` in our previous version. The previous design only has `Create run`,  popping up with the creation modal allowing the DS to choose the run type(one-off or scheduled). Probably we also need` View schedules` since we have `View runs`. 
<img width="872" alt="Screenshot 2024-02-28 at 12 03 22" src="https://github.com/opendatahub-io/odh-dashboard/assets/10469467/d4f3963e-1fb0-45b5-8a01-6b7c3ecc74cc">

Reply:
@jpuzz0 maybe we log a quick little story to chase down after this to add buttons to the details page and effectively anywhere there is a "Create run" that is not explicitly on the run page. And we can wrap this PR up but still address Yan's feedback. How does that sound? Alternatively you could do it to this PR but I think you're just about ready to go

Reply:
If we want to remove the run type in the creation modal, I would suggest to change the` Create run` button under the `Active run tab` to a dropdown and include a second option `Schedule run` to open the Schedule run modal, because runs under the Run tab are not only created by immediate run, but also from the execution results of schedules. Under the Schedule tab, I would suggest to change the `Create schedule` to "`Schedule run` to be consistent.

Reply:
This JIRA is meant to capture some of these UX changes: https://issues.redhat.com/browse/RHOAIENG-3887.

Reply:
follow up jira created for this thread.

 https://issues.redhat.com/browse/RHOAIENG-3887